Makefile

```diff
@@ -9,7 +9,7 @@ MAKE_ENV_FILE = local.vars.mk
 
 # Setup the secret required for observability then generate and apply the files
 setup-observability: acm/odh-core/acm-observability/kustomization.yaml
-	kustomize build acm/odh-core/acm-observability | kubectl apply -f -
+	oc kustomize acm/odh-core/acm-observability | oc apply -f -
```

Comment:
I guess this one could alternatively be `oc apply -k acm/odh-core/acm-observability`

Reply:
Updated, rebased on `main`. Sorry it took me a while to get back to this.